[DOCS] Update how to create a checkpoint with Test YAML config (#7835)
great-expectations/great_expectations: docs/docusaurus/docs/guides/validation/checkpoints/how_to_configure_a_new_checkpoint_using_test_yaml_config.md

import Prerequsities from '../../connecting_to_your_data/components/prerequisites.jsx'
import TechnicalTag from '@site/docs/term_tags/_tag.mdx';

> This how-to guide demonstrates advanced examples for configuring a <TechnicalTag tag="checkpoint" text="Checkpoint" /> using `test_yaml_config`. **Note:** For a basic guide on creating a new Checkpoint, please see [How to create a new Checkpoint](../../../guides/validation/checkpoints/how_to_create_a_new_checkpoint.md).

Reviewer: ```suggestion
Use the information provided here to learn how to configure a <TechnicalTag tag="checkpoint" text="Checkpoint" /> using `test_yaml_config`. To create a new Checkpoint,  see [How to create a new Checkpoint](../../../guides/validation/checkpoints/how_to_create_a_new_checkpoint.md).
```
![image](https://user-images.githubusercontent.com/104205257/236825148-c2a52340-2946-4153-9138-6e974f5d32e6.png)

> `test_yaml_config` is a convenience method for configuring the moving parts of a Great Expectations deployment. It allows you to quickly test out configs for <TechnicalTag tag="datasource" text="Datasources" />, <TechnicalTag tag="store" text="Stores" />, and Checkpoints. `test_yaml_config` is primarily intended for use within a notebook, where you can iterate through an edit-run-check loop in seconds.

Reviewer: ```suggestion
`test_yaml_config` lets you configure and test the components of a Great Expectations deployment, including <TechnicalTag tag="datasource" text="Datasources" />, <TechnicalTag tag="store" text="Stores" />, and Checkpoints. `test_yaml_config` is intended for use within a Jupyter notebook, where you can use an edit-run-check loop to quickly test your changes before implementation.
```
If you don't implement the suggested edit, change _convenience_ to _convenient_.

</Prerequisites>

## Overview

`test_yaml_config` supports iteratively testing configuration to help zeroing in on the Checkpoint configuration you want. As an iterative workflow, it is particularly well suited to notebooks.

### 1. Setup

> This code in your first cell will load the necessary modules and initialize your <TechnicalTag tag="data_context" text="Data Context"/>:

Reviewer: ```suggestion
Run the following command in the first cell to load the necessary modules and initialize your <TechnicalTag tag="data_context" text="Data Context"/>:
```

```python name="tests/integration/docusaurus/validation/checkpoints/how_to_configure_a_new_checkpoint_using_test_yaml_config.py setup"
```

> ### 2. Listing Assets

Reviewer: ```suggestion
### 2. List Assets
```
![image](https://user-images.githubusercontent.com/104205257/236829046-34397de0-d6e8-4d74-9bcb-c465ec325fe9.png)

> Your Checkpoint configuration will bring together Data Assets and Expectation Suites. The following code will list the available asset names:

Reviewer: ```suggestion
Your Checkpoint configuration includes Data Assets and Expectation Suites. Run the following command to list the available asset names:
```

```python name="tests/integration/docusaurus/validation/checkpoints/how_to_configure_a_new_checkpoint_using_test_yaml_config.py asset_names"
```

> This code will list the available Expectation Suites:

Reviewer: ```suggestion
Run the following command to list the Expectation Suites:
```

```python name="tests/integration/docusaurus/validation/checkpoints/how_to_configure_a_new_checkpoint_using_test_yaml_config.py suite_names"
```

### 3. Creating your Checkpoint

> The following YAML (inline as a Python string) defines a `SimpleCheckpoint` as a starting point:

Reviewer: ```suggestion
Run the following YAML (inline as a Python string) to define `SimpleCheckpoint` as the starting point:
```
I'm not clear what is meant by _starting point_. Can this be clarified?

```python name="tests/integration/docusaurus/validation/checkpoints/how_to_configure_a_new_checkpoint_using_test_yaml_config.py create_checkpoint"
```

> ### 4. Testing your Checkpoint Configuration

Reviewer: ```suggestion
### 4. Test your Checkpoint Configuration
```
![image](https://user-images.githubusercontent.com/104205257/236828962-6c700c07-7d53-4dbc-bd79-260b9df1925b.png)

> Once you have your YAML configuration, you can test it to make sure it's correct.

Reviewer: ```suggestion
When you have your YAML configuration, run the following command to test it and ensure it's correct:
```

```python name="tests/integration/docusaurus/validation/checkpoints/how_to_configure_a_new_checkpoint_using_test_yaml_config.py test_checkpoint"
```

> Tweaking the above configuration and testing with the above cell will help ensure your configuration changes are correct.

Reviewer: ```suggestion
Modifying the previous configuration and testing with the previous cell helps ensure your configuration changes are correct.
```

> ### 5. Saving your Checkpoint

Reviewer: ```suggestion
### 5. Save your Checkpoint
```
![image](https://user-images.githubusercontent.com/104205257/236829865-35097e9a-e840-4623-a450-f701b7d8db5b.png)

> To save your Checkpoint, add it to the Data Context:

Reviewer: ```suggestion
Run the following command to save your Checkpoint and add it to the Data Context:
```
